### ISSUE TITLE
The app fail to update

### ISSUE THREAD

**Issue by osenvosem** [notable/notable#577]
- **OS Version**: MacOS 10.14.3
- **Notable Version**: 1.3.0

### Current behavior

Doesn't update.

### Expected behavior

Expect the app to update.

**Comment by fabiospampinato** [comment]
What happens if you click on the menu option "Notable -> Check for Updates..."?

**Comment by osenvosem** [comment]
This happens, again and again.
<img width="340" alt="Screenshot 2019-03-22 at 21 26 08" src="https://user-images.githubusercontent.com/6762673/54845557-aec95880-4cea-11e9-84a9-f802b34cafb5.png">

**Comment by fabiospampinato** [comment]
@osenvosem Thanks, could you check what files the `/Users/srg/Library/Application Support/Caches/notable-updater` directory contains? I think we might have shipped with a broken version of the updater 🤔 That's not good, this things have to be reliable.

**Comment by osenvosem** [comment]
```
.
└── pending

1 directory, 0 files
```

**Comment by fabiospampinato** [comment]
And is there anything inside the `pending` directory?

**Comment by osenvosem** [comment]
No, it's empty. Sorry for not being clear enough. :)

**Comment by fabiospampinato** [comment]
Ok the contrary, thanks for the help!

**Comment by fabiospampinato** [comment]
@semiopat  can you give more details about your system? like what OS version are you using? and where is the Notable.app located?

**Comment by semiopat** [comment]
I'm on macOS Mojave 10.14.4, and Notable is located in the Applications folder.

**Comment by fabiospampinato** [comment]
@semiopat 🤔 I'm really not sure why this is happening, do you have some firewall set up or anything weird going on?

**Comment by semiopat** [comment]
No, I have no firewall, the udapte process simply crashes at the end, after downloading the files of the update. But surprise: I cannot reproduce it anymore, I just launch the app and it appears to have silently update, finally. That was not the case this morning and the days before.

**Comment by fabiospampinato** [comment]
According to some comments I've been reading online it seems like this error may be triggered when trying to download the update multiple times simultaneously, so we might have to insert a lock or something to prevent that.

I'll take a closer look at this, in the meantime I've opened an upstream bug report: https://github.com/electron-userland/electron-builder/issues/4590
